.eslintrc.js

```diff
@@ -8,9 +8,7 @@ NodePlugin.RULES_DIR = path.resolve(__dirname, 'tools', 'eslint-rules');
 
 const ModuleFindPath = Module._findPath;
 const hacks = [
-  'eslint-plugin-node-core',
-  'eslint-plugin-markdown',
-  'babel-eslint',
+  'eslint-plugin-node-core'
```

Comment:
The other two on npm, so I installed them using internal npm.

Reply:
https://www.npmjs.com/package/eslint-plugin-node-core has not been updated properly (though it's probably just a matter of running the update script again)

Reply:
I found that too. Perhaps just leave this plugin in this repo for now.